api/hostmanagement/openamt/openamt.go

```diff
@@ -32,7 +36,7 @@ func NewService(dataStore portainer.DataStore) *Service {
 	}
 	return &Service{
 		httpsClient: &http.Client{
-			Timeout: time.Second * time.Duration(5),
+			Timeout: time.Second * time.Duration(60),
 			Transport: &http.Transport{
 				TLSClientConfig: &tls.Config{InsecureSkipVerify: true},
```

Comment:
Should InsecureSkipVerify be configurable?  do we always want this on?

Reply:
Yep, this pretty needs to be set to true or I get this error on any request:`x509: cannot validate certificate for 198.199.100.44 because it doesn't contain any IP SANs)`.

I stole it from the GIT API service https://github.com/portainer/portainer/blob/develop/api/git/git.go#L119